### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by PhobiaCide** [Incomplete-Infinity/discussions#3] [General]
This is a general discussion.

**Comment by HobbyDungeon** [comment]
![image](https://github.com/Incomplete-Infinity/discussions/assets/121633459/db2f8e29-6a9a-4a2b-abbc-360417529364)

**Reply by HobbyDungeon** [reply]
![image](https://github.com/Incomplete-Infinity/discussions/assets/121633459/f9207d03-6e63-4511-af9e-47f130109284)

**Reply by HobbyDungeon** [reply]
![image](https://github.com/Incomplete-Infinity/discussions/assets/121633459/b46ed5eb-e35a-4320-bdf5-1e34cd913932)

**Reply by HobbyDungeon** [reply]
![image](https://github.com/Incomplete-Infinity/discussions/assets/121633459/c9546bdc-53b6-49e1-ae4c-b5fc96f9aab2)

**Reply by HobbyDungeon** [reply]
![image](https://github.com/Incomplete-Infinity/discussions/assets/121633459/76027ee9-877c-486a-81e5-74002c791b4e)

**Reply by HobbyDungeon** [reply]
![image](https://github.com/Incomplete-Infinity/discussions/assets/121633459/ea0bb2ad-6b2d-4e0e-805b-bce1563d12ad)

**Reply by HobbyDungeon** [reply]
![image](https://github.com/Incomplete-Infinity/discussions/assets/121633459/d6fa8eab-8e26-4f28-ae8c-93063bd20dbf)

**Reply by HobbyDungeon** [reply]
![image](https://github.com/Incomplete-Infinity/discussions/assets/121633459/f436d7a2-0776-4397-a84a-3c0df0dafd43)

**Reply by HobbyDungeon** [reply]
![image](https://github.com/Incomplete-Infinity/discussions/assets/121633459/debf77f6-e4f9-4de9-a2b8-9dcc0ab96ffd)

**Reply by HobbyDungeon** [reply]
![image](https://github.com/Incomplete-Infinity/discussions/assets/121633459/58d9d006-ea81-4f9f-add1-00212fe33ede)

**Reply by HobbyDungeon** [reply]
![image](https://github.com/Incomplete-Infinity/discussions/assets/121633459/8c12fe24-a2ed-4f5d-bcda-71a820655db5)

**Reply by PhobiaCide** [reply]
1. All of the bars are simple progress indicators
2. The glowing orbs could be checkboxes or radio buttons

Try using the words `Bento Box` in your prompts. It's the current trend.

**Reply by PhobiaCide** [reply]
The World Map reminds me of an idea I had for a world map that indicated day and night with Eve buddies' locations pinned to it. Also applicable to real-world team coordination.

**Reply by PhobiaCide** [reply]
This background is awesome!

**Comment by PhobiaCide** [comment]
> ![image](https://private-user-images.githubusercontent.com/121633459/303504871-f436d7a2-0776-4397-a84a-3c0df0dafd43.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg4NjAwMjgsIm5iZiI6MTcwODg1OTcyOCwicGF0aCI6Ii8xMjE2MzM0NTkvMzAzNTA0ODcxLWY0MzZkN2EyLTA3NzYtNDM5Ny1hODRhLTNjMGRmMGRhZmQ0My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyNVQxMTE1MjhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MzAyOGEyNDY2YWVlNDRmOTE4YzU2Nzk5M2I0ZmNkNGZhZGJiODljNDlkZmFjZjZkNWNmMTZhODQ3Yjc2YmM3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.b0PswXSPNTQ7IzGJTbiSLFiJ5jeRtaHrIhNitrTLhog)

Can you try to generate some backgrounds like this? That's probably the best place to start.
